charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
 version: 1.1.16
```

Comment:
Need to increment the chart version.

Reply:
Updated here - https://github.com/aws/amazon-vpc-cni-k8s/pull/1995 

branch protection prevented my update in this PR.